modules/cli/src/main/scala/scala/cli/commands/package0/PackageOptions.scala

```diff
@@ -49,10 +49,11 @@ final case class PackageOptions(
   @Group(HelpGroup.Package.toString)
   @HelpMessage("Generate a source JAR rather than an executable JAR")
   @Name("sources")
-  @Name("src")
+  @Name("source")
+  @Tag(tags.deprecated("source")) // alias to be removed in 1.6.x
   @Tag(tags.restricted)
   @Tag(tags.inShortHelp)
-    source: Boolean = false,
+    src: Boolean = false,
```

Comment:
maybe we should actually be more verbose here? `--outputSources` or `--jarSources`. `src` seems too short and it also can be mixed up with `src` used for naming of the sources directory and easily misused. Like `--src ../additional-sources` for example.

Reply:
`--src` is an existing alias, I merely now made it the default name... no need to remove it IMO.
I can add `--jar-sources` as an extra alias, if needed.

Reply:
added `--jar-sources` and `--sources-jar` as additional aliases.